shared/src/commonMain/kotlin/com/vml/tutorial/plantshop/basket/presentation/components/BasketScreen.kt

```diff
@@ -0,0 +1,283 @@
+                    .fillMaxWidth(),
+                horizontalArrangement = Arrangement.SpaceBetween,
+                verticalAlignment = Alignment.CenterVertically
+            ){
+                Text(
+                    item.plant.priceText,
+                    style = Typography.headlineSmall,
+                    color = Color.Black.copy(alpha = 0.6f),
+                )
+                QuantityController(
+                    value = item.quantity,
+                    onQuantityChanged = { changedValue ->
+                        onQuantityChanged.invoke(item.plant.id, changedValue)
+                    }
+                )
+            }
+        }
+    }
+}
+
+@Composable
+private fun BasketSummary(
+    state: BasketScreenState,
+    modifier: Modifier = Modifier,
+    onCheckout: () -> Unit,
+) {
+    Column(
+       modifier = modifier
+    ) {
+        SummaryListItem(
+            title = UiText.StringRes(MR.strings.basket_summary_items_title).asString(),
+            value = state.itemsPriceText
+        )
+        SummaryListItem(
+            title = UiText.StringRes(MR.strings.basket_summary_discount_title).asString(),
+            value = state.discountText
+        )
+        Divider(
+            modifier = Modifier
+                .padding(horizontal = 4.dp, vertical = 10.dp),
+            thickness = 1.dp,
+            color = Color.Gray.copy(alpha = 0.1f)
+        )
+        Checkout(
+            totalPrice = state.totalPriceText,
+            modifier = Modifier
+                .fillMaxWidth(),
+            onCheckout = onCheckout
+        )
+    }
+}
+
+@Composable
+private fun SummaryListItem(
+    title: String,
+    value: String,
+    modifier: Modifier = Modifier
+) {
+    Row(
+        modifier = modifier
+            .height(35.dp)
+            .fillMaxWidth(),
+        horizontalArrangement = Arrangement.SpaceBetween
+    ) {
+        Text(
+            title,
+            style = Typography.titleMedium,
+            fontWeight =FontWeight.Normal,
+            color = Color.Gray.copy(alpha = 0.6f),
+        )
+        Text(
+            value,
+            style = Typography.titleMedium,
+            color = Color.Black.copy(alpha = 0.6f),
+        )
+    }
+}
+
+@Composable
+private fun Checkout(
+    totalPrice: String,
+    modifier: Modifier = Modifier,
+    onCheckout: () -> Unit
+) {
+    Row(
+        modifier = modifier,
+        horizontalArrangement = Arrangement.SpaceBetween
+    ) {
+        Column {
+            Text(
+                UiText.StringRes(MR.strings.basket_summary_total_price_title).asString(),
+                style = Typography.labelMedium,
+                fontWeight =FontWeight.Normal,
+                color = Color.Gray.copy(alpha = 0.6f),
+            )
+            Text(
+                totalPrice,
+                style = Typography.headlineSmall,
+                color = Color.Black.copy(alpha = 0.6f),
+            )
+        }
+        Button(
```

Comment:
I think the checkout button would look better if it was centered in relation to the combined height of the 'Total Price' text and the sum underneath

Reply:
Optical illusion 😄